webapp/actions/websocket_actions.jsx

```diff
@@ -241,6 +245,12 @@ function handleUserRemovedEvent(msg) {
     }
 }
 
+function handleUserUpdatedEvent(msg) {
+    if (UserStore.getCurrentId() !== msg.user_id) {
+        AsyncClient.getProfiles();
```

Comment:
yeah, I thought the same thing but since this shouldn't occur that often then I thought It's fine, since this is actually the behaviour when a new user is added.

I could send the user info with the event and update the info for that user in `UserStore` and then fire an `onChange` event.